src/appshell/view/firstlaunchsetup/firstlaunchsetupmodel.cpp

```diff
@@ -123,4 +125,8 @@ bool FirstLaunchSetupModel::askAboutClosingEarly()
 void FirstLaunchSetupModel::finish()
 {
     configuration()->setHasCompletedFirstLaunchSetup(true);
+
+    if (!interactive()->isOpened(HOME_PAGE_URI).val) {
+        interactive()->open(HOME_PAGE_URI);
```

Comment:
Not completely sure about this one. I don't think we will want to open the home page unconditionally. For example, the user might have launched MuseScore for the first time by opening a MSCZ file. In that case, the user wants to see the Score tab after finishing the first launch setup.

Reply:
Oh ok. Well, I did it because whenever I build and run MuseScore in Qt Creator and I close the window, it takes me to the score page when home page would have been better for me